src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, Dispatch, SetStateAction } from 'react';
+import { Divider, List, ListItem, ListItemText, ListItemIcon, Box, CircularProgress, ListSubheader } from '@mui/material';
+import { Link } from 'react-router-dom';
+
+import useStyles from './styles';
+
+import githubWhite from '../assets/logo/github_white.png';
+// import githubblack from '../assets/logo/github_black.png';
+
+const demoMain = [{ label: 'Pull Request', value: 'pull-request' },
```

Comment:
Please name this const what it is used for - naming convention
`demoMain` implies some kind of demonstration, not menu items.

Reply:
Fixed - I have renamed it to mainMenu and the other one to otherMenu as the name of the submenus shows on the design